public/app/features/inspector/styles.ts

```diff
@@ -32,7 +33,6 @@ export const getPanelInspectorStyles2 = (theme: GrafanaTheme2) => {
     content: css`
       flex-grow: 1;
       height: 100%;
-      overflow: scroll;
```

Comment:
this was added in https://github.com/grafana/grafana/pull/47832 but all scrolling all inspect tabs works without it